src/common/paths/users_userID_password.yml

```diff
@@ -18,6 +18,82 @@ post:
 
     #### Related guides
 
+    - [InfluxDB Cloud - Change your password](https://docs.influxdata.com/influxdb/cloud/account-management/change-password/)
+    - [InfluxDB OSS - Change your password](https://docs.influxdata.com/influxdb/latest/users/change-password/)
+  parameters:
+    - $ref: '../parameters/TraceSpan.yml'
+    - in: path
+      name: userID
+      schema:
+        type: string
+      required: true
+      description: The ID of the user to set the password for.
+  requestBody:
+    description: The new password to set for the user.
+    required: true
+    content:
+      application/json:
+        schema:
+          $ref: '../schemas/PasswordResetBody.yml'
+  responses:
+    '204':
+      description: Success. The password is updated.
+    '400':
+      description: |
+        Bad request.
+
+        #### InfluxDB Cloud
+
+        - Doesn't allow you to manage passwords through the API; always responds with this status.
+
+        #### InfluxDB OSS
+
+        - Doesn't understand a value passed in the request.
+      content:
+        application/json:
+          schema:
+            $ref: "../schemas/Error.yml"
+          examples:
+            updatePasswordNotAllowed:
+               summary: Cloud API can't update passwords
+               value: {
+                        "code": "invalid",
+                        "message": "passwords cannot be changed through the InfluxDB Cloud API"
+                      }
+    default:
+      description: Unexpected error
+      $ref: '../responses/ServerError.yml'
+  x-codeSamples:
+    - lang: Shell
+      label: "cURL: use Basic auth to update the user password"
+      source: |
+        curl --request POST \
+           "http://localhost:8086/api/v2/users/USER_ID/password" \
+          --header 'Content-type: application/json' \
+          --user "USERNAME:PASSWORD" \
+          --data-binary @- << EOF
+            {"password": ""}
+        EOF
+put:
+  operationId: PutUsersIDPassword
+  tags:
+    - Security and access endpoints
+    - Users
+  summary: Update a password
+  description: |
+    Updates a user password.
+
+    Use this endpoint to let a user authenticate with
```

Comment:
Based on the above, this section is likely incorrect.

Reply:
> Based on the above, this section is likely incorrect.

True, `PUT` with Basic Auth doesn't seem to work.

Reply:
Description is updated to:

```
Updates a user password.

    #### InfluxDB Cloud
```